tests/api.spec.ts

```diff
@@ -5,6 +5,7 @@ import * as _ from "lodash";
 import * as sinon from "sinon";
 import { IAction, ActionType } from "../src/actions/interfaces";
 import { ActionsController } from '../src/actions/controller';
+import * as vscode from "vscode";
```

Comment:
Please remove.
We are using --> mockVscode

Reply:
it's impossible to do set on mockVscode